virttest/storage.py

```diff
@@ -179,6 +184,10 @@ def get_image_filename(params, root_dir, basename=False):
             ceph_monitor = params.get('ceph_monitor')
             return ceph.get_image_filename(ceph_monitor, rbd_pool_name,
                                            rbd_image_name, ceph_conf)
+        if enable_nvme:
+            address = params.get('nvme_pci_address')
```

Comment:
It seems to be mandatory param so suggest `params['nvme_pci_address']`

Reply:
Thanks, have updated.